prefixdb_test.go

```diff
@@ -20,6 +26,143 @@ func mockDBWithStuff(t *testing.T) DB {
 	return db
 }
 
+func taskKey(i, k int) []byte {
+	return []byte(fmt.Sprintf("task-%d-key-%d", i, k))
+}
+
+func randomValue() []byte {
+	b := make([]byte, 16)
+	_, err := rand.Read(b)
+
+	if err != nil {
+		return nil
+	}
+
+	return b
+}
+
+func TestWithGolevelDB(t *testing.T) {
+	dir := t.TempDir()
+	path := filepath.Join(dir, "goleveldb")
+
+	db, err := NewGoLevelDB(path, "")
+	require.NoError(t, err)
+
+	t.Run("GolevelDB", func(t *testing.T) { Run(t, db) })
+}
```

Comment:
Since these are split out by backend, we can skip the additional indirection through the subtests in each of these (the top-level test function will already be named in the output anyway), e.g.:
```suggestion
func TestGolevelDB(t *testing.T) {
	path := filepath.Join(t.TempDir(), "goleveldb")

	db, err := NewGoLevelDB(path, "")
	require.NoError(t, err)

	Run(t, db)
}
```